### PR TITLE
IBX-6852: Focus mode described

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-6852)
| Versions      | master

Focus mode described.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by lucasOsti — APPROVED** [review]

**Review by adriendupuis — APPROVED** [review]
https://ez-systems-developer-documentation--242.com.readthedocs.build/projects/userguide/en/242/content_management/create_edit_content_items/#focus-mode is nicelly illustrated.

The "Edit Rich Text Fields" section is unbalanced with more details and screenshots about the focus mode than for the text formatting toolbar usage. Detailing the toolbar could be a future enhancement of this section.

File: docs/content_management/img/focus_mode.png
Comment (MagdalenaZuba) [inline docs/content_management/img/focus_mode.png:1]: Can you change the screenshot to shorter with less white space?

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Comment by julitafalcondusza** [comment]
> https://ez-systems-developer-documentation--242.com.readthedocs.build/projects/userguide/en/242/content_management/create_edit_content_items/#focus-mode is nicely illustrated.
> 
> The "Edit Rich Text Fields" section is unbalanced with more details and screenshots about the focus mode than for the text formatting toolbar usage. Detailing the toolbar could be a future enhancement of this section.

Agree, unfortunately it's not in the scope of this PR to improve "Edit Rich Text Fields" section, but we need to think about it. :)

**Comment by MagdalenaZuba** [comment]
> > https://ez-systems-developer-documentation--242.com.readthedocs.build/projects/userguide/en/242/content_management/create_edit_content_items/#focus-mode is nicely illustrated.
> > The "Edit Rich Text Fields" section is unbalanced with more details and screenshots about the focus mode than for the text formatting toolbar usage. Detailing the toolbar could be a future enhancement of this section.
> 
> Agree, unfortunately it's not in the scope of this PR to improve "Edit Rich Text Fields" section, but we need to think about it. :)

Can you create jira ticket for that? It's easy to forget those things when you move to other projects :)

**Comment by julitafalcondusza** [comment]
> > > https://ez-systems-developer-documentation--242.com.readthedocs.build/projects/userguide/en/242/content_management/create_edit_content_items/#focus-mode is nicely illustrated.
> > > The "Edit Rich Text Fields" section is unbalanced with more details and screenshots about the focus mode than for the text formatting toolbar usage. Detailing the toolbar could be a future enhancement of this section.
> > 
> > 
> > Agree, unfortunately it's not in the scope of this PR to improve "Edit Rich Text Fields" section, but we need to think about it. :)
> 
> Can you create jira ticket for that? It's easy to forget those things when you move to other projects :)

Will do :)

**Review by MagdalenaZuba — APPROVED** [review]

**Comment by dabrt** [comment]
> https://ez-systems-developer-documentation--242.com.readthedocs.build/projects/userguide/en/242/content_management/create_edit_content_items/#focus-mode is nicely illustrated.
> 
> The "Edit Rich Text Fields" section is unbalanced with more details and screenshots about the focus mode than for the text formatting toolbar usage. Detailing the toolbar could be a future enhancement of this section.

Adding more screenshots equals having to update them when the interface changes ever so slightly.
I'd rather have less illustrations and better descriptions in the text.

I guess this is a decision we'd have to make at a certain point - how should we approach this,

File: docs/content_management/create_edit_content_items.md [hunk 1]
```diff
@@ -120,6 +120,23 @@ To add a new element to the Field, choose one of the available elements:
 Each of these elements can have its own settings, such as text formatting.
 The option bar also lets you reorder or remove any elements in the Rich Text Field.
 
+#### Focus mode
+
+While editing Rich Text Fields, you can switch to focus mode.
+
+Focus mode expandes the workspace to full screen and shows only editor toolbar.
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:127]:
```suggestion
Focus mode expands the workspace to full screen and shows only the editor toolbar.
```

**Review by dabrt — APPROVED** [review]
One minor change required

**Pull request merged**